.readthedocs.yaml

```diff
@@ -0,0 +1,31 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+# build:
+#   os: ubuntu-22.04
```

Comment:
I think we probably want to default this on, so it's more explicit? Otherwise it will just use the default image, and perhaps break in the future.